smaps/src/main/webapp/index.html

```diff
@@ -54,6 +54,11 @@
     <div class="row justify-content-md-center">
         <p id="error-message" name="error-message"></p>
     </div>
+    <div class="row justify-content-md-center">
+        <form action="/fileupload" method="POST" enctype="multipart/form-data">
+            <input type="submit" name="example-btn" id="example-btn" value="EXAMPLE MODE">
```

Comment:
Consider naming all your control surfaces with some consistent style.

E.g.,
"submit"
  vs
"example-btn"

You may want to rename "submit" to "submit-btn" in another change.

Reply:
Good idea, I'll change that.